build.sbt

```diff
@@ -194,28 +194,28 @@ lazy val commonJsLibSettings = gspScalaJsSettings ++ commonLibSettings ++ Seq(
 )
 
 lazy val commonWDS = Seq(
-  version in webpack := "4.43.0",
-  version in startWebpackDevServer := "3.11.0",
+  webpack / version := "4.43.0",
+  startWebpackDevServer / version := "3.11.0",
   webpackConfigFile in fastOptJS := Some(
```

Comment:
Should we change this to fastOptJS / webpackConfigFile too? (And the same for fullOptJS?)

Reply:
Great catch